src/main/java/org/launchcode/liftoffrecipetracker/controllers/RecipeController.java

```diff
@@ -54,54 +59,73 @@ public String displayRecipes(@RequestParam(required = false) Integer categoryId,
 				model.addAttribute("title", "Recipes with Beverage: " + beverage.getName());
 				model.addAttribute("recipes", beverage.getRecipes());
 			}
-		}
-		return "recipe/index";
-	}
 
-	@GetMapping("create")
-	public String displayCreateRecipeForm(Model model) {
-		model.addAttribute("title", "Create a Recipe");
-		model.addAttribute("categories", categoryRepository.findAll());
-		model.addAttribute("beverages", beverageRepository.findAll());
-		model.addAttribute(new Recipe());
-		return "recipe/create";
+		} else if (userSession == null) {
+			User user = authenticationController.getUserFromSession(userSession);
+				model.addAttribute("title", "Recipes with Owners: " + user.getUsername());
+				model.addAttribute("recipes", user.getRecipes());
+			}
```

Comment:
I don't think we ever get to this block of code. I think you can take it out or we can talk about how you can accomplish seeing all of the recipes by a user!